tests/h/services/group_create_test.py

```diff
@@ -471,13 +473,3 @@ def svc(db_session, usr_svc, publish):
 @pytest.fixture
 def creator(factories):
     return factories.User(username="group_creator")
-
-
-class GroupScopeWithOrigin(Matcher):
-    """Matches any GroupScope with the given origin."""
-
-    def __init__(self, origin):
-        super().__init__(
-            f"* any group with origin: {origin} *",
-            lambda other: isinstance(other, GroupScope) and other.origin == origin,
-        )
```

Comment:
This wasn't strictly a coverage issue, but the same code was causing problems elsewhere so I scooped it out